package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Zooniverse",
   "license": "Apache-2.0",
   "dependencies": {
-    "json-api-client": "^4.0.2",
+    "json-api-client": "file:../json-api-client",
```

Comment:
I'm going to revert this change. I had trouble getting the right json-api-client loaded up with PFE when testing locally. Using npm link with both PJC and this dep got the right version of everything loaded for manual testing for me.

Reply:
Actually no, the combination of this and then npm linking them both to PFE is what was working. This will need to be reverted for merge and publishing though.

Reply:
I think we'll need a release candidate of https://github.com/zooniverse/json-api-client/pull/52 to test this properly. What's the best of of dealing this that?

Reply:
I've published json-api-client@5.0.0-rc.0 and have updated the dependency here.